.changeset/silent-taxis-peel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@leafygreen-ui/code': minor
```

Comment:
@DesignerDave this work? (i'm not familiar w the leafygreen release system)

Reply:
Yep, perfect!